database-population.js

```diff
@@ -0,0 +1,72 @@
+
+let arrayOfUsers = [];
+
+const generateRandomNumber = function (val) {
+    return Math.ceil(val * Math.random());
+}
+
+const userGeneration = function () {
+    for (let i = 0; i < numUsers; i++) {
+        let tempObject = {};
+        tempObject.userId = i;
+        tempObject.name = faker.name.findName();
+        tempObject.avgRating = generateRandomNumber(5);
+        // will break in the case where Math.random() returns 1 - not sure if - 0.01 will work
+        tempObject.picture = arrayOfAvatarURLs[Math.floor((arrayOfAvatarURLs.length - 0.01) * Math.random())];
+        arrayOfUsers.push(tempObject);
+    }
+}
+
+let arrayOfReviews = [];
+
+const reviewGeneration = function () {
+    for (let i = 0; i < numReviews; i++) {
+        let tempObject = {};
+        tempObject.houseId = generateRandomNumber(numUsers);
+        tempObject.reviewTitle = faker.lorem.sentence();
+        tempObject.reviewText = faker.lorem.paragraph();
+        tempObject.reviewDate = faker.date.past();
+        tempObject.helpfulCount = generateRandomNumber(10);
+        tempObject.reportedCount = {inappropriate: generateRandomNumber(10), hateful: generateRandomNumber(10), fake: generateRandomNumber(10)}
+        tempObject.response = {name: faker.name.findName(), responseText: faker.lorem.sentences(), responseDate: faker.date.past(), responsePicture: arrayOfAvatarURLs[Math.floor((arrayOfAvatarURLs.length - 0.01) * Math.random())]};
+        tempObject.rating = {overall: generateRandomNumber(5), interior: generateRandomNumber(5), exterior: generateRandomNumber(5), location: generateRandomNumber(5), noise: generateRandomNumber(5)};
+        // 1 is solo, 2 is group, 3 is couple, 4 is family
+        tempObject.group = generateRandomNumber(4);
+        tempObject.user = Math.floor(99 * Math.random());
+        arrayOfReviews.push(tempObject);
+    }
+}
+
+userGeneration();
+
+console.log('Array of Users: ', arrayOfUsers);
+
+const populateUsers = function () {
+    db.User.insertMany(arrayOfUsers, function (error, docs) {
+        if (error) console.log(error);
+        console.log(docs);
+    });
+}
+
+populateUsers();
+
+reviewGeneration();
+
+const populateReviews = function () {
+    db.Review.insertMany(arrayOfReviews, function (error, docs) {
+        if (error) console.log(error);
+        console.log(docs);
+    });
+}
+
+populateReviews();
+
+//tell node to terminate 
+// process.exit(-1);
```

Comment:
Remove comments here...though I think it would be a good idea to run a process.exit() when all the db work is done. This is probably easiest done if you use promises for the insertMany calls and do all of your inserts in one function...let me know if you're interested in learning how to do this.